setup.py

```diff
@@ -66,7 +65,6 @@ def find_version(*file_paths):
         'sqlite': ["sqlalchemy"],
     },
     dependency_links=[
-        "git+https://github.com/ODM2/geoalchemy.git@v0.7.3#egg=geoalchemy-0.7.3",
         "git+https://github.com/ODM2/ODM2PythonAPI@v0.1.0-alpha#egg=odm2api-0.1.0"
```

Comment:
Question: should `odm2api ` be specified with a pinned version here? I guess it does, but wanted to confirm.